src/CodingMilitia.PlayBall.Shared.StartupTasks/StaticExtensions.cs

```diff
@@ -0,0 +1,13 @@
+using CodingMilitia.PlayBall.Shared.StartupTasks;
+using Microsoft.EntityFrameworkCore;
+
+namespace Microsoft.Extensions.DependencyInjection
+{
+    public static class StaticExtensions
```

Comment:
Just nitpicking, but call it `ServiceCollectionExtensions`, I think it's what usually done, unless there are other extensions for `IServiceCollection` and we want to have them separated.

Reply:
hmmmm. I think we already have that class. 
https://github.com/AspNetCoreFromZeroToOverkill/GroupManagement/blob/2f64d5c0f0be7b63dabed58b722da185079e623d/src/CodingMilitia.PlayBall.GroupManagement.Web/IoC/ServiceCollectionExtensions.cs#L11

My initial idea is this would be used as a class with static shared methods. Or do we transfer the whole `ServiceCollectionExtension` to the `StartupTasks` project?

Reply:
We have that class in one project, we can have one with the same name in another one.
We won't even have a need to disambiguate the names, as we use them for extension methods, we never reference them by name 😉

Reply:
Renamed the class now 😁